src/supremm/plugins/CpuUsage.py

```diff
@@ -94,12 +94,29 @@ def computeallcpus(self):
     def computejobcpus(self):
         """ stats for the cores on the nodes that were assigend to the job (if available) """
 
-        proc = self._job.getdata('proc')
-
-        if proc == None:
+        # Check both proc and procprom preprocessors for CPU information
+        proc = None
+        for preproc in ('proc', 'procprom'):
```

Comment:
The architectural design is that the plugins should be agnostic to the data source wherever possible. In this case does the CPU plugin really need to know whether the data came from prometheus or pcp? It appears not given that the code that uses the cpusallowed data is unchanged.

Please revert this change to the cpu plugin and just have the prometheus preprocessor use 'proc' as the identifier.

Reply:
These changes have been reverted in https://github.com/ubccr/supremm/pull/259/commits/dd2ec42481bc4e9cd2bb5e5dc76427262ed4f8b2